src/assets/main.scss

```diff
@@ -49,6 +49,7 @@
   --warm-grey: #767676;
   --footer-grey: #ebebeb;
   --primary-blue: #0079ff;
+  --secondary-blue: #0056b3;
```

Comment:
I called this "secondary blue" but we can change the name, this is just a color for the default blue of the <a> hover that we use. Since our <a> tag under the CTA button contains an SVG element (the icon), the default browser behavior is not to underline images inside of the <a> tags. I had to mimic that behavior with a border-bottom under the <a> tag.